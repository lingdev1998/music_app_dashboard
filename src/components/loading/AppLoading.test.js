import React from 'react';
import { shallow } from 'enzyme';  
import AppLoading from './appLoading';

describe('<AppLoading />', () => {
    const AppLoadingWrapper = shallow(<AppLoading/>);
    it('AppLoadingWrapper matches snapshot', () => {
        expect(AppLoadingWrapper).toMatchSnapshot()
    })
})