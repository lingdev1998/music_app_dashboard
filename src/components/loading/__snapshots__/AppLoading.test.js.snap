// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLoading /> AppLoadingWrapper matches snapshot 1`] = `ShallowWrapper {}`;
